Classes/Domain/Model/ZoomApiAccessToken.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace CodeQ\ZoomApi\Domain\Model;
+
+class ZoomApiAccessToken
+{
+    public readonly string $accessToken;
+    public readonly array $scope;
```

Comment:
`composer.json` requires `"php": "^8"` (so including 8.0), but `readonly` props are only available starting with 8.1 - so either we convert this to regular private props that are only set-able via constructor and have accessors, or we require 8.1

If we go the 8.1 route, we could also make use of [constructor property promotion with readonly props](https://stitcher.io/blog/php-81-readonly-properties):

```php
class ZoomApiAccessToken
{
    public function __construct(
        public readonly string $accessToken,
        public readonly array $scope,
    ) {}
}
```